handler.go

```diff
@@ -25,6 +25,8 @@ const (
 // ClusterID we will use when talking to clients.
 var ClusterID = "dekafclusterid"
 
+type MaxOffsetFn func(time.Time) int64
```

Comment:
nit: Add a comment explaining the intended semantics and purpose of a `MaxOffsetFn`?

Reply:
Added a comment!